### PR TITLE
Fixes typo and renames multi_process_utility

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

**Comment by hanefi** [comment]
I like how you fixed the typo, but I am not ok with renaming `multi_ProcessUtility`. There is a history behind that name, and I do not see any value in renaming it.

**Comment by gurkanindibay** [comment]
> I like how you fixed the typo, but I am not ok with renaming `multi_ProcessUtility`. There is a history behind that name, and I do not see any value in renaming it.



> I like how you fixed the typo, but I am not ok with renaming `multi_ProcessUtility`. There is a history behind that name, and I do not see any value in renaming it.

@onurctirtir said it is better to change the name. multi does not make any sense to me. Could you describe what multi is? About the multiple database? if it is , it is very indirect way to emphasize this. it is not intuitive

File: src/backend/distributed/commands/utility_hook.c
Comment (onurctirtir) [inline src/backend/distributed/commands/utility_hook.c:1]: can we also rename ProcessUtilityInternal to citus_ProcessUtilityInternal?

**Review by onurctirtir — APPROVED** [review]
I agree with @hanefi in the sense that it might not add much value for current members of the team but I also see value in doing this for new contributors.

**Pull request merged**